polynote-frontend/polynote/state/notebook_state.ts

```diff
@@ -235,6 +254,26 @@ export class NotebookStateHandler extends StateHandler<NotebookState> {
             }, this)
         })
     }
+
+    get isLoading(): boolean {
```

Comment:
We also set the loading state in `ServerStateHandler`. You can call `ServerStateHandler.getNotebook(this.path)?.loaded` to get the value.